build.cmd

```diff
@@ -79,6 +79,8 @@ set TEST_END_2_END=0
 set INCLUDE_TEST_SPEC_NUNIT=
 set INCLUDE_TEST_TAGS=
 
+set COPY_FSCOMP_RESOURCE_FOR_BUILD_FROM_SOURCES=0
```

Comment:
@KevinRansom can you check if you like this and if it is OK? Thanks!

Reply:
@smoothdeveloper this is something that we need to fix in buildfromsource, although, this seems like a pretty decent work around.

The reason the copy is needed is that the buildtask used for the RedHat build doesn't yet or at least, didn't the last time we looked support the resource id generation.

@brettfo are we in a position to update this yet?

Reply:
@KevinRansom thanks, yes if in the future the build from source can prepare the package by doing the copy for us from the canonical files at their current place, it would remove the duplicates and some noise in each of the changes to FSComp.

The other pain area is the localization files, it is great that they are open source and fully integrated in the delivery process, but we should avoid the need to check them in this repository, or at least have them automatically updated in our CI, there can be another localization CI which doesn't make our PR here red and is handled by Redmond for the actual release pipeline.

Hope that makes sense, thanks all for reviewing.